lib/api/controllers/writeController.js

```diff
@@ -31,6 +31,9 @@ module.exports = function WriteController (kuzzle) {
         if (rooms.length > 0) {
           kuzzle.notifier.notify(rooms, new ResponseObject(requestObject).toJson());
         }
+        else if (!requestObject.isPersistent()) {
+          kuzzle.notifier.notify({}, new ResponseObject(requestObject), context.connection);
+        }
```

Comment:
Why that ?
it seems not related to authorization checking.... 
Was there a bug with kuzzle response there ?